### PR TITLE
Add e2e encryption content for pulsar

### PR DESCRIPTION
**Pull request by yaron2**
Closes https://github.com/dapr/docs/issues/3366.

### CODE REVIEW

File: daprdocs/content/en/reference/components-reference/supported-pubsub/setup-pulsar.md [hunk 1]
```diff
@@ -115,6 +118,90 @@ curl -X POST http://localhost:3500/v1.0/publish/myPulsar/myTopic?metadata.delive
       }'
 ```
 
+### E2E Encryption
+
+Dapr supports setting public and private key pairs to enable Pulsar's [end-to-end encryption feature](https://pulsar.apache.org/docs/3.0.x/security-encryption/)
```
Comment (hhunter-ms) [inline daprdocs/content/en/reference/components-reference/supported-pubsub/setup-pulsar.md]:
```suggestion
Dapr supports setting public and private key pairs to enable Pulsar's [end-to-end encryption feature](https://pulsar.apache.org/docs/3.0.x/security-encryption/).
```

File: daprdocs/content/en/reference/components-reference/supported-pubsub/setup-pulsar.md [hunk 2]
```diff
@@ -115,6 +118,90 @@ curl -X POST http://localhost:3500/v1.0/publish/myPulsar/myTopic?metadata.delive
...
+    value: "localhost:6650"
+  - name: publicKey
+    value: ./public.key
+  - name: privateKey
+    value: ./private.key
+```
+
+#### Enabling publisher encryption from value
+
+*Note: It is recommended to reference the public key from a secret as [shown here]({{< ref component-secrets.md >}})*
```
Comment (hhunter-ms) [inline daprdocs/content/en/reference/components-reference/supported-pubsub/setup-pulsar.md]:
```suggestion
> Note: It is recommended to [reference the public key from a secret]({{< ref component-secrets.md >}}).
```

File: daprdocs/content/en/reference/components-reference/supported-pubsub/setup-pulsar.md [hunk 3]
```diff
@@ -115,6 +118,90 @@ curl -X POST http://localhost:3500/v1.0/publish/myPulsar/myTopic?metadata.delive
...
+    value: "localhost:6650"
+  - name: publicKey
+    value:  "-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA1KDAM4L8RtJ+nLaXBrBh\nzVpvTemsKVZoAct8A+ShepOHT9lgHOCGLFGWNla6K6j+b3AV/P/fAAhwj82vwTDd\nruXSflvSdmYeFAw3Ypphc1A5oM53wSRWhg63potBNWqdDzj8ApYgqjpmjYSQdL5/\na3golb36GYFrY0MLFTv7wZ87pmMIPsOgGIcPbCHker2fRZ34WXYLb1hkeUpwx4eK\njpwcg35gccvR6o/UhbKAuc60V1J9Wof2sNgtlRaQej45wnpjWYzZrIyk5qUbn0Qi\nCdpIrXvYtANq0Id6gP8zJvUEdPIgNuYxEmVCl9jI+8eGI6peD0qIt8U80hf9axhJ\n3QIDAQAB\n-----END PUBLIC KEY-----\n"
+  - name: keys
+    value: myapp.key
+```
+
+#### Enabling consumer encryption from value
+
+*Note: It is recommended to reference the public and private keys from a secret as [shown here]({{< ref component-secrets.md >}})*
```
Comment (hhunter-ms) [inline daprdocs/content/en/reference/components-reference/supported-pubsub/setup-pulsar.md]:
```suggestion
> Note: It is recommended to [reference the public and private keys from a secret]({{< ref component-secrets.md >}}).
```

**Review by hhunter-ms — DISMISSED** [review]
tiny review

**Comment by yaron2** [comment]
> tiny review

Fixed.

**Review by msfussell — APPROVED** [review]
LGTM

**Pull request merged**